integrations-and-supported-tools/skorch/notebooks/Neptune_Skorch.ipynb

```diff
@@ -0,0 +1,531 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "device = 'cuda' if torch.cuda.is_available() else 'cpu'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mnist_dim = X.shape[1]\n",
+    "hidden_dim = int(mnist_dim/8)\n",
+    "output_dim = len(np.unique(mnist.target))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mnist_dim, hidden_dim, output_dim"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Neural network in PyTorch's framework."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ClassifierModule(nn.Module):\n",
+    "    def __init__(\n",
+    "            self,\n",
+    "            input_dim=mnist_dim,\n",
+    "            hidden_dim=hidden_dim,\n",
+    "            output_dim=output_dim,\n",
+    "            dropout=0.5,\n",
+    "    ):\n",
+    "        super(ClassifierModule, self).__init__()\n",
+    "        self.dropout = nn.Dropout(dropout)\n",
+    "\n",
+    "        self.hidden = nn.Linear(input_dim, hidden_dim)\n",
+    "        self.output = nn.Linear(hidden_dim, output_dim)\n",
+    "\n",
+    "    def forward(self, X, **kwargs):\n",
+    "        X = F.relu(self.hidden(X))\n",
+    "        X = self.dropout(X)\n",
+    "        X = F.softmax(self.output(X), dim=-1)\n",
+    "        return X"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Initialize Neptune run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Connect your script to Neptune application and create new run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = neptune.init_run(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,\n",
+    "    project='common/skorch-integration',\n",
+    "    name='skorch-example'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You tell Neptune: \n",
+    "\n",
+    "* **who you are**: your Neptune API token `api_token` \n",
+    "* **where you want to send your data**: your Neptune `project`.\n",
+    "\n",
+    "At this point you have new run in Neptune. For now on you will use `run` to log metadata to it.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "\n",
+    "Instead of logging data to the public project `'common/skorch-integration'` as an anonymous user 'neptuner' you can log it to your own project.\n",
```

Comment:
```suggestion
    "Instead of logging data to the public project 'common/skorch-integration' as an anonymous user 'neptuner' you can log it to your own project.\n",
```